hazelcast/protocol/codec/count_down_latch__await_codec.py

```diff
@@ -0,0 +1,29 @@
+from hazelcast.serialization.bits import *
```

Comment:
Can you remove this file from the PR? I see that it is unrelated to aggregators.